<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePostWindow</class>
 <widget class="QMainWindow" name="CreatePostWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="user_groupBox">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>275</y>
      <width>451</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="create_pushButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>280</y>
       <width>89</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>30</y>
       <width>351</width>
       <height>231</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="post_title_label">
          <property name="text">
           <string>Post Title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="post_title_lineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="post_text_label">
          <property name="text">
           <string>Post Text</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="post_text_lineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="visibility_label">
          <property name="text">
           <string>Post Privacy</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="public_radioButton">
            <property name="text">
             <string>public</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="private_radioButton">
            <property name="text">
             <string>private</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
